scripts/mktcli.js

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env babel-node
+
+import { eth, Log, cli, contracts } from 'decentraland-commons'
+import { loadEnv } from './utils'
+import { decodeAssetId, encodeAssetId } from './monitor/utils'
+
+const log = new Log('mktcli')
+
+const logError = err => log.error('ERR: ' + err.message)
+const parseCoords = coords =>
+  coords
+    .replace('(', '')
+    .replace(')', '')
+    .split(',')
+
+const landRegistry = () => eth.getContract('LANDRegistry')
+const marketplace = () => eth.getContract('Marketplace')
+
+const main = {
+  addCommands(program) {
+    program.command('decode').action(async options => {
+      if (!options.length) return
+
+      try {
+        const coords = await decodeAssetId(options)
+        log.info(`(decode) str:${options} => coords:(${coords})`)
+      } catch (err) {
+        logError(err)
+      }
+    })
+
+    program.command('encode').action(async options => {
+      if (!options.length) return
+
+      try {
+        const [x, y] = parseCoords(options)
+        const assetId = await encodeAssetId(x, y)
+        log.info(
+          `(encode) coords:(${x},${y}) => str:${assetId.toString()} hex:${assetId.toString(
+            16
+          )}`
+        )
+      } catch (err) {
+        logError(err)
+      }
+    })
+
+    program.command('land-owner').action(async options => {
+      if (!options.length) return
+
+      try {
+        const [x, y] = parseCoords(options)
+        const contract = landRegistry()
+        const owner = await contract.ownerOfLand(x, y)
```

Comment:
maybe we could add an `if Contract.isEmptyAddress() => search if it's a district` ?